pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -43,11 +43,20 @@ type PermissionsProvider interface {
 	GetUserPermissions(ctx context.Context, query GetUserPermissionsQuery) ([]*Permission, error)
 }
 
-type PermissionsServices interface {
-	GetTeamService() PermissionsService
-	GetFolderService() PermissionsService
-	GetDashboardService() PermissionsService
-	GetDataSourceService() PermissionsService
+type TeamPermissionsService interface {
+	SetUserPermission(ctx context.Context, orgID int64, user User, resourceID, permission string) (*ResourcePermission, error)
```

Comment:
Yes it is but the only thing we will use for team permissions is SetUserPermissions and maybe GetPermissions. So that is the only thing we should expose in the interface